inc/agent/communication/abstractrequest.class.php

```diff
@@ -279,7 +279,11 @@ public function addError($message, $code = 500) {
       if ($this->headers->hasHeader('GLPI-Agent-ID')) {
          $this->addToResponse([
             'status' => 'error',
-            'message' => $message,
+            'message' => preg_replace(
+               '|\$ref\[file~2///var/www/html/private/glpi/vendor/glpi-project/inventory_format/inventory.schema.json\]|',
```

Comment:
Using a hard-coded path here is wrong.
```suggestion
               '|\$ref\[file~2//.*/vendor/glpi-project/inventory_format/inventory.schema.json\]|',
```